apps/studio/src/features/dashboard/components/ResourceTable/ResourceTableMenu.tsx

```diff
@@ -69,13 +69,15 @@ export const ResourceTableMenu = ({
               Edit folder settings
             </MenuItem>
           )}
-          <MenuItem
-            as="button"
-            onClick={handleMoveResourceClick}
-            icon={<BiFolderOpen fontSize="1rem" />}
-          >
-            Move to...
-          </MenuItem>
+          {(type === "Page" || type === "Folder") && (
```

Comment:
just to check we do have different types right? Like CollectionPage - are they movable?

Reply:
No, they are not. The others are Collection, CollectionPage and Rootpage for completeness